src/main/java/org/candlepin/subscriptions/event/EventController.java

```diff
@@ -361,6 +362,12 @@ private void ensureOptIn(String orgId) {
   }
 
   public Event normalizeEvent(Event event) {
+    // NOTE we will probably remove the below serviceType normalization
+    // after https://issues.redhat.com/browse/SWATCH-2533
+    // placeholder card to remove it in https://issues.redhat.com/browse/SWATCH-2794
+    if (Objects.nonNull(event.getServiceType()) && event.getServiceType().equals(AAP_CLUSTER)) {
+      event.setServiceType("Ansible Managed Node");
+    }
```

Comment:
super hack :)
I don't really like this, but as you already pointed out, there is a plan to remove it.